addie/processing/mantid/master_table/master_table_loader.py

```diff
@@ -231,6 +232,8 @@ def load(self):
             _target_row_entry["normalization"] = self._retrieve_element_dict(element='Normalization',
                                                                              source_row_entry=_source_row_entry)
 
+            _target_row_entry["align_and_focus_args"] = _source_row_entry["AlignAndFocusArgs"]
```

Comment:
Talked offline but we should switch this to a `_source_row_entry.get("AlignAndFocusArgs")` and handle when `None` returned to `_target_row_entry["align_and_focus_args"]`.